dlog/plonk-5-wires/src/range.rs

```diff
@@ -0,0 +1,17 @@
+use algebra::Field;
+use std::ops::Range;
+
+pub const PSDN  : Range<usize> = 0..5;
```

Comment:
This seems error-prone; it would be nice if we could infer this automatically from the gates that we use for a particular constraint system.